File: src/Services/WareHouse/WareHouse.API/Application/Model/OutwardDetailDTO.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace WareHouse.API.Application.Model
{
    public partial class OutwardDetailDTO : BaseModel
    {
        public OutwardDetailDTO()
        {
            SerialWareHouses = new HashSet<SerialWareHouseDTO>();
        }

        public string OutwardId { get; set; }
        public string ItemId { get; set; }
        public string ItemName { get; set; }
        public string Code { get; set; }
        public string UnitName { get; set; }

        public string UnitId { get; set; }
        public decimal Uiquantity { get; set; }
        public decimal Uiprice { get; set; }
        public decimal Amount { get; set; }
        public decimal Quantity { get; set; }
        public decimal Price { get; set; }
        public string DepartmentId { get; set; }
        public string DepartmentName { get; set; }
        public string EmployeeId { get; set; }
        public string EmployeeName { get; set; }
        public string StationId { get; set; }
        public string StationName { get; set; }
        public string ProjectId { get; set; }
        public string ProjectName { get; set; }
        public string CustomerId { get; set; }
        public string CustomerName { get; set; }
        public string AccountMore { get; set; }

        public string AccountYes { get; set; }

        public virtual WareHouseItemDTO Item { get; set; }
        public virtual OutwardDTO Outward { get; set; }
        public virtual UnitDTO Unit { get; set; }
        public virtual ICollection<SerialWareHouseDTO> SerialWareHouses { get; set; }

        public virtual IEnumerable<WareHouseItemDTO> WareHouseItemDTO { get; set; }
        public virtual IEnumerable<UnitDTO> UnitDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetDepartmentDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetEmployeeDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetStationDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetProjectDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetCustomerDTO { get; set; }
        public virtual IEnumerable<BaseSelectDTO> GetAccountDTO { get; set; }
        public virtual InwardDTO Inward { get; set; }
    }
}

File: src/WebClient/WareHouse/src/app/model/OutwardDetailDTO.ts
import { BaseModel } from "./BaseModel";
import { BaseSelectDTO } from "./BaseSelectDTO";
import { InwardDTO } from "./InwardDTO";
import { SerialWareHouseDTO } from "./SerialWareHouseDTO";
import { UnitDTO } from "./UnitDTO";
import { WareHouseItemDTO } from "./WareHouseItemDTO";

export interface OutwardDetailDTO extends BaseModel {
    outwardId: string;
    itemId: string;
    unitId: string;
    uiquantity: number;
    uiprice: number;
    amount: number;
    quantity: number;
    price: number;
    departmentId: string;
    departmentName: string;
    employeeId: string;
    employeeName: string;
    stationId: string;
    stationName: string;
    projectId: string;
    projectName: string;
    customerId: string;
    customerName: string;
    accountMore: string;
    accountYes: string;
    status: string;
    serialWareHouses: SerialWareHouseDTO[];
    outward: InwardDTO|null;
    item: WareHouseItemDTO|null;
    unit: UnitDTO|null;
    unitDTO: UnitDTO[];
    wareHouseItemDTO: WareHouseItemDTO[];
    getDepartmentDTO: BaseSelectDTO[];
    getEmployeeDTO: BaseSelectDTO[];
    getStationDTO: BaseSelectDTO[];
    getProjectDTO: BaseSelectDTO[];
    getCustomerDTO: BaseSelectDTO[];
    getAccountDTO: BaseSelectDTO[];
    //BaseSelectDTO
   // serialWareHouses: SerialWareHouse[];
}